src/partials/footer.html

```diff
@@ -59,7 +59,7 @@
 
   <!-- Further information -->
   <div class="md-footer-meta md-typeset">
-    <div class="md-footer-meta__inner md-grid">
+    <div class="md-footer-meta__inner">
```

Comment:
This extra change is just to align the copyright information to the left of the footer (like we have on the landing page):

![image](https://user-images.githubusercontent.com/60105800/137959934-cb581352-1c3e-4c79-9003-4143052bd566.png)